server/test/unit/plugins/file/test_model_distributor.py

```diff
@@ -0,0 +1,256 @@
+import errno
+import os
+import shutil
+import tempfile
+import unittest
+
+from mock import Mock, MagicMock, patch
+
+from pulp.devel.mock_distributor import get_publish_conduit
+from pulp.plugins.file.distributor import FilePublishProgressReport
+from pulp.plugins.file.model_distributor import FileDistributor, BUILD_DIRNAME
+from pulp.server.db.model import Repository
+
+
+DATA_DIR = os.path.realpath("../../../data/")
+SAMPLE_RPM = 'pulp-test-package-0.3.1-1.fc11.x86_64.rpm'
+SAMPLE_FILE = 'test-override-pulp.conf'
+
+
+class FileDistributorTest(unittest.TestCase):
+    """
+    Tests the file distributor base class
+    """
+    def setUp(self):
+        self.temp_dir = tempfile.mkdtemp()
+
+        self.target_dir = os.path.join(self.temp_dir, "target")
+        self.repo = MagicMock(spec=Repository)
+        self.repo.id = "foo"
+        self.repo.working_dir = self.temp_dir
+        self.unit = MagicMock(unit_type_id='RPM', name=SAMPLE_RPM, size=1, checksum='sum1',
+                              unit_key={'name': SAMPLE_RPM, 'checksum': 'sum1', 'size': 1})
+        self.unit.storage_path = os.path.join(__file__, DATA_DIR, SAMPLE_RPM)
+        self.publish_conduit = get_publish_conduit(existing_units=[self.unit, ])
```

Comment:
Maybe this is a style thing and not a PEP8 guideline I can't remember, but you don't need the comma here.